Source/Core/DolphinQt/AboutDialog.ui

```diff
@@ -19,67 +19,125 @@
   <property name="windowTitle">
    <string>About Dolphin</string>
   </property>
-  <layout class="QVBoxLayout" name="verticalLayout">
-   <item>
-    <widget class="QLabel" name="label">
+  <layout class="QGridLayout" name="gridLayout">
+   <item row="5" column="1">
+    <widget class="QLabel" name="lblLicenseAuthorsSupport">
      <property name="text">
-      <string>&lt;big&gt;&lt;b&gt;Dolphin&lt;/b&gt;&lt;/big&gt; %1&lt;br&gt;
-© 2003-%2 Dolphin Team&lt;br&gt;
-Branch: %3&lt;br&gt;
-Revision: %4&lt;br&gt;
-Compiled: %5 @ %6&lt;br&gt;
-Dolphin is a GameCube/Wii emulator, which was originally written by&lt;br&gt;F|RES and ector. Today Dolphin is an open source project with many&lt;br&gt; contributors, too many to list. If interested, just go check out &lt;a href=&quot;https://github.com/dolphin-emu/dolphin&quot;&gt;the project&lt;br&gt;page&lt;/a&gt;.&lt;br&gt;
-&lt;br&gt;
-Special thanks to Bushing, Costis, CrowTRobo, Marcan, Segher, Titanik,&lt;br&gt;or9, and Hotquik for their reverse engineering and docs/demos.&lt;br&gt;
-&lt;br&gt;
-Big thanks to Gilles Mouchard whose Microlib PPC emulator gave our&lt;br&gt; development a kickstart.&lt;br&gt;
-&lt;br&gt;
-Thanks to Frank Wille for his PowerPC disassembler, which or9 and we&lt;br&gt; modified to include Gekko specifics.&lt;br&gt;
-&lt;br&gt;
-Thanks to hcs/destop for their GC ADPCM decoder.&lt;br&gt;
-&lt;br&gt;
-We are not affiliated with Nintendo in any way. GameCube and Wii are&lt;br&gt; trademarks of Nintendo. This emulator should not be used to play games&lt;br&gt;you do not legally own.</string>
+      <string>&lt;a href=&quot;https://github.com/dolphin-emu/dolphin/blob/master/license.txt&quot;&gt;License&lt;/a&gt; | &lt;a href=&quot;https://github.com/dolphin-emu/dolphin/graphs/contributors&quot;&gt;Authors&lt;/a&gt; | &lt;a href=&quot;https://forums.dolphin-emu.org/&quot;&gt;Support&lt;/a&gt;</string>
```

Comment:
Just the 3 individual words, and keep the rest away from the translators. I don't see how reordering 3 more or less unrelated words makes sense, other than personal preference.
Plus, if they do it wrong, it could either be broken, or worse not have the links at all. And who knows what the open source licensing gods will do if someone dares to remove the "License" link.